Sources/Commands/SwiftTool.swift

```diff
@@ -766,7 +766,8 @@ private func getEnvBuildPath() -> AbsolutePath? {
     // Don't rely on build path from env for SwiftPM's own tests.
     guard POSIX.getenv("IS_SWIFTPM_TEST") == nil else { return nil }
     guard let env = POSIX.getenv("SWIFTPM_BUILD_DIR") else { return nil }
-    return AbsolutePath(env, relativeTo: localFileSystem.currentWorkingDirectory!)
+    guard let currentWorkingDirectory = localFileSystem.currentWorkingDirectory else { return nil }
```

Comment:
We should move `currentWorkingDirectory` to be a parameter to `getEnvBuildPath()` method.

Reply:
@aciidb0mb3r do you think okay this parameter naming?